.github/workflows/ci.yml

```diff
@@ -4,7 +4,7 @@ on:
   push:
     branches: [ master ]
   pull_request:
-    branches: [ master ]
+    branches: [ master, integrate-optimized-native-forms-branch]
```

Comment:
@syedowaisali90 Please remember to revert this before merging optimize branch to master

Reply:
sure